src/navigation/routeKeys.ts

```diff
@@ -26,6 +26,7 @@ export const treasuryScreen = 'Treasury' as const;
 export const motionsScreen = 'Motions' as const;
 export const democracyScreen = 'Democracy' as const;
 export const referendumScreen = 'Referendum' as const;
+export const democracyProposalScreen = 'Proposal' as const;
```

Comment:
It would be better to set the title to the screen instead of using this as default name, If we add a screen for Treasury Proposal for example we would have conflicts.

```suggestion
export const democracyProposalScreen = 'DemocracyProposal' as const;
```